scalalib/src/mill/javalib/android/AndroidLintModule.scala

```diff
@@ -0,0 +1,78 @@
+package mill.javalib.android
+
+import mill._
+import mill.api.PathRef
+import mill.javalib.android.AndroidAppModule
+
+/**
+ * Android Lint Module for integrating the Android Lint tool in a Mill build.
+ *
+ * This module provides configuration options for executing Android Lint, including setting paths,
+ * specifying lint rules, managing reports, and more.
+ */
+@mill.api.experimental
+trait AndroidLintModule extends AndroidAppModule {
```

Comment:
what is the added value to have it as a dedicated trait and not a part of the `AndroidAppModule`?